### DISCUSSION TITLE
Report article

### DISCUSSION THREAD

**Discussion by sschneider-ihre-pvs** [dailydotdev/daily#487] [Feature Requests]
It would be cool if there were some reporting options for like, outdated, broken link and so on

**Comment by idoshamun** [comment]
Hi @sschneider-ihre-pvs,

We actually have it already. Please click on the three dots button on the card and you should see a context menu with more options.
See below
<img width="362" alt="image" src="https://user-images.githubusercontent.com/1993245/155871345-f4d6db99-af1a-4aaa-a096-6cea370c0ff0.png">
